pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -203,7 +203,7 @@ func createCSISecret(clientset kubernetes.Interface, namespace string, clusterIn
 		Type: k8sutil.RookType,
 	}
 	k8sutil.SetOwnerRef(&csiSecret.ObjectMeta, ownerRef)
-	if _, err = clientset.CoreV1().Secrets(namespace).Create(csiSecret); err != nil {
+	if _, err = context.Clientset.CoreV1().Secrets(namespace).Create(csiSecret); err != nil {
```

Comment:
You need to update the secret if it already exists, right?

Reply:
Yes, it needs to be changed by the newly generated key, let me do this.